File: 3-Semestre/Java e C#/ATs/C--AT-1T/Questao06/Aluno.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AT_C_.Questao06
{
    internal class Aluno
    {
        public string Nome { get; set; }
        public string Matricula { get; set; }
        public string Curso { get; set; }
        public double MediaNotas { get; set; }

        public Aluno(string nome, string matricula, string curso, double mediaNotas)
        {
            Nome = nome;
            Matricula = matricula;
            Curso = curso;
            MediaNotas = mediaNotas;
        }

        public void ExibirDados()
        {
            Console.WriteLine("=== Dados do Aluno ===");
            Console.WriteLine($"Nome: {Nome}");
            Console.WriteLine($"Matrícula: {Matricula}");
            Console.WriteLine($"Curso: {Curso}");
            Console.WriteLine($"Média das Notas: {MediaNotas:F2}");
            Console.WriteLine($"Situação: {VerificarAprovacao()}");
        }

        public string VerificarAprovacao()
        {
            return MediaNotas >= 7 ? "Aprovado" : "Reprovado";
        }

        public static void Executar()
        {
            Aluno aluno1 = new Aluno("Guilherme Reis", "2024001", "Engenharia de Software", 8.5);

            aluno1.ExibirDados();
        }
    }
}

File: 3-Semestre/Java e C#/ATs/AT-JAVA-1T/src/main/java/org/example/Questao07/Aluno.java
package org.example.Questao07;

public class Aluno {

    private String nome;
    private String matricula;
    private double nota1, nota2, nota3;

    public Aluno(String nome, String matricula, double nota1, double nota2, double nota3) {
        this.nome = nome;
        this.matricula = matricula;
        this.nota1 = nota1;
        this.nota2 = nota2;
        this.nota3 = nota3;
    }

    public double calcularMedia() {
        return (nota1 + nota2 + nota3) / 3;
    }

    public void verificarAprovacao() {
        double media = calcularMedia();
        System.out.println("Média: " + media);
        if (media >= 7) {
            System.out.println("Foi: Aprovado");
        } else {
            System.out.println("Foi: Reprovado");
        }
    }
}

